openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/EntityRepository.java

```diff
@@ -265,7 +266,12 @@ public void restorePatchAttributes(T original, T updated) {
 
   /** Set fullyQualifiedName of an entity */
   public void setFullyQualifiedName(T entity) {
-    entity.setFullyQualifiedName(entity.getName());
+    entity.setFullyQualifiedName(EntityInterfaceUtil.quoteName(entity.getName()));
+  }
+
+  /** Set fullyQualifiedNameHash of an entity */
+  public String getFullyQualifiedNameHash(T entity) {
+    return FullyQualifiedName.buildHash(entity.getFullyQualifiedName());
```

Comment:
We are unnecessarily overriding this method in several places even though the overriding methods have same implementation.